R/s3_methods.R

```diff
@@ -80,131 +88,230 @@
 }
 
 #' @export
-#' @noRd
+#' @rdname S3_subset
 `[.LazyFrame` = `[.DataFrame`
 
+#' Take the first n rows
+#'
+#' @param x A Polars DataFrame or LazyFrame
+#' @param n Number of rows
+#' @param ... Not used
+#'
 #' @export
-#' @noRd
+#' @rdname S3_head
 head.DataFrame = function(x, n = 6L, ...) x$limit(n = n)
 
 #' @export
-#' @noRd
+#' @rdname S3_head
 head.LazyFrame = head.DataFrame
 
+#' Take the last n rows
+#'
+#' @param x A Polars DataFrame or LazyFrame
```

Comment:
```suggestion
#' @param x A [DataFrame][DataFrame_class] or [LazyFrame][LazyFrame_class]
```

How about linking to the classes?
Like this.

https://github.com/pola-rs/r-polars/blob/103f95601df3842b7d017517ea4a0b752c22f74a/R/sql.R#L64

Reply:
I'll apply this but not that the links will only appear in RStudio, not on the website

Reply:
Yes, but the link should be inserted in the generated Rd file rather than in RStudio.
I think it will be properly linked in the documentation on R-universe.